### PR TITLE
Remove unused extras from all languages.

### PR DESCRIPTION
**Pull request by samscott89**

### CODE REVIEW

File: languages/js/test/parity.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ class A {
 }
 oso.registerClass(A);
 
-class D extends A {}
+class D extends A { }
```
Comment (gj) [inline languages/js/test/parity.ts:19]: I don't think your editor agrees with our linter setup.

Comment (samscott89) [reply languages/js/test/parity.ts]: 🤔

**Review by gj — COMMENTED** [review]
Remove these too?

https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/languages/python/oso/tests/parity/policies/test_api.polar#L9

https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/languages/python/oso/tests/parity/test_api_externals.py#L9-L50

**Review by gj — COMMENTED** [review]
https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/docs/examples/application/expenses-flask/app/authorization.py#L4

**Comment by samscott89** [comment]
> 
> 
> Remove these too?
> 
> https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/languages/python/oso/tests/parity/policies/test_api.polar#L9
> 
> https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/languages/python/oso/tests/parity/test_api_externals.py#L9-L50

I intentionally kept these / moved them into the Python test suite as part of https://github.com/osohq/oso/pull/571 since I think they do test some reasonably interesting behaviour.

**Comment by gj** [comment]
https://github.com/osohq/oso/blob/50830cb59485afb96e576611f6932a0b57cef836/docs/examples/application/expenses-spring-boot/src/main/java/com/example/springboot/Authorizer.java#L4

File: docs/changelogs/vNEXT.rst [hunk 2]
```diff
@@ -28,6 +28,16 @@ authorization instead of the removed
...
 
+
+Removed "extras"
+----------------
+
+The oso library previously had some additional default supported classes:
+``Http`` and ``Pathmapper``. These have been deprecated and removed.
+
+Instead, you should register suitable classes directly. For example,
+registering the ``werkzeug.wrappers.Request`` object for Flask applications.
```
Comment (gj) [inline docs/changelogs/vNEXT.rst:39]: Looks like we already (conditionally) register the `flask.Request` class (a subclass of `werkzeug.wrappers.Request`) automatically in the `flask-oso` lib.

**Review by gj — APPROVED** [review]
Ship it and I'll update the changelog on the 0.9.0 branch

File: docs/changelogs/vNEXT.rst (hunk 2 above)
Comment (samscott89) [reply docs/changelogs/vNEXT.rst:39]: Hmm, that's a good point. One of the reasons for removing Http is that you probably just want whatever the framework uses.

Comment (samscott89) [reply docs/changelogs/vNEXT.rst]: Updated.

**Pull request merged**